daffodil-core/src/main/scala/org/apache/daffodil/grammar/ModelGroupGrammarMixin.scala

```diff
@@ -71,5 +73,7 @@ trait ModelGroupGrammarMixin
     finalContent
   }
 
+  private lazy val groupContent1 = schemaSet.sharedGroupContentsFactory.getShared(shareKey, groupContent)
```

Comment:
Same question as with groupMembers1, can we find a better name?

Reply:
I am doing a few renames here, but in general this will be groupContent and protected groupContentDef following the same pattern as for ModelGroup.groupMembers.

The _content will become groupContentWithInitiatorTerminator since that's what is is.